apps/academy/src/data/questions/lesson-5/1-intro/Q4.json

```diff
@@ -0,0 +1,18 @@
+{
+  "question": "How does a web browser function in the front-end of a Web3 app?",
```

Comment:
i think this question needs rethinking. the focus on the browser feels off; i think you're just getting at "what is the front-end of a web3 application responsible for?" - but it likely contains html too, so the first answer is relevant too. if you want to gut check alternatives, maybe just relay them in discord for faster feedback cycles.

Reply:
Agree